dannielRolfe/file-1.js

```diff
@@ -0,0 +1,255 @@
+//ecxercise #1
```

Comment:
Nice job reviewing your work! Feel free to get in touch if you have any questions about any of this, particularly the ones that you've self-marked as incorrect.